rtil/src/native/livesplit.rs

```diff
@@ -0,0 +1,49 @@
+use std::sync::Mutex;
+use livesplit_core::{Segment, TimeSpan, Timer, TimingMethod};
+use once_cell::sync::Lazy;
+
+static LIVESPLIT_STATE: Lazy<Mutex<LiveSplit>> = Lazy::new(|| Mutex::new(LiveSplit::new()));
+
+pub struct LiveSplit {
+    pub run: livesplit_core::Run,
+    pub timer: Timer,
+}
+pub struct _Timer {}
```

Comment:
Don't name this `_Timer`. Don't start stuff with underscores (unless it's an unused variable which in fact shouldn't be used). When importing something via `use`, you can rename it: `use livesplit_core::Timer as LsTimer`. Or just refer to it via `livesplit_core::Timer` like you're doing with `Run` (though I'd probably prefer `LsRun` or `LiveSplitRun` as well).